### PR TITLE
Update plum_lightpad docs to mention flow config

### PR DESCRIPTION
**Pull request by prystupa**
## Proposed change

Update docs to mention config flow

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/36802
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by balloob** [comment]
THis PR needs rebasing off the current branch.

**Comment by prystupa** [comment]
> THis PR needs rebasing off the current branch.

Thanks, I just rebased

File: source/_integrations/plum_lightpad.markdown [hunk 1]
```diff
@@ -20,7 +22,8 @@ Configurable/Dimmable Wi-Fi Lightswitch
 
 ## Configuration
 
-To enable Plum Lightpad support, add the following lines to your `configuration.yaml` file:
+To use Plum Lightpad devices in your installation, add your Plum account from the integrations page.
```
Comment (frenck) [inline source/_integrations/plum_lightpad.markdown:25]: Please describe to the user what to do. See examples of other integrations in our documentation

File: source/_integrations/plum_lightpad.markdown [hunk 2]
```diff
@@ -20,7 +22,8 @@ Configurable/Dimmable Wi-Fi Lightswitch
...
+Alternatively, add the following lines to your `configuration.yaml` file:
```
Comment (frenck) [inline source/_integrations/plum_lightpad.markdown:32]: Isn't the config deprecated? If so, we should remove it.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/plum_lightpad.markdown (hunk 2 above)
Comment (prystupa) [reply source/_integrations/plum_lightpad.markdown:32]: Hi Frenk. The config is still being supported by this integration, so adding config flow is transparent to pre-existing users. Do you want me to keep this reference in this case? Or is it the best practice to remove support from being documented?

Comment (frenck) [reply source/_integrations/plum_lightpad.markdown:32]: Normally we deprecate YAML config if it was fully replaced with frontend.
But if it isn't deprecated, then it is fine to leave it in.

File: source/_integrations/plum_lightpad.markdown (hunk 1 above)
Comment (prystupa) [reply source/_integrations/plum_lightpad.markdown:25]: thanks, updated!

**Review by frenck — APPROVED** [review]
Thanks, @prystupa 👍

**Pull request merged**